tools/privacyidea-decrypt-safeword.py

```diff
@@ -30,7 +30,7 @@
             current_token["serial"] = serial
 
         if re.match("sccTokenData", line):
-            m = re.search("sccKey=\((.*?)\).*", line)
+            m = re.search("sccKey=\((.*?)\).*?sccSeq=\((.*?)\).*", line)
```

Comment:
adding a `raw` string specifier here removes the PEP8 warning

Reply:
done in b302d64d